### ISSUE TITLE
Feature Request: QA from seq-tools

### ISSUE THREAD

**Issue by rosibaj** [icgc-argo/seq-tools#67]
1. `INVALID` should actually mean an invalid payload that we do not want submitted. Currently, the “c670_coverage_estimate” check is run and assigned a value of `PASS` or `INVALID`,  but is actually optional.  It can fail, but the user can still submit that data based on user-judgement of the data.  This is more of a fyi or warning status.  If the submitter has more data they can add it, but they may not and choose to still submit.
- [x] **Remove this check.**  I discussed adding this as a warning on Slack, but in discussion with Christina, said its better to just remove it until we know more

2. The report result is long and adds unactionable (thus unnecessary) information, making it harder to read when debugging. Overall the experience could be better in helping me identify actionable items to make my payloads valid. 
Anytime a validation fails, I have to read through everything that passes to get to the failures.
- [x] To make it more actionable, it should only present the failures.  the `INVALID` and `PASS` metrics could be separated into two reports for this. 
- [x] Failures are in individual reports, meaning I have to go through many folder to get to issues. It would be nice to see a whole report, of all failures so i could work with many at once.  For example: 
```
{"summary": 
"INVALID": 4, 
{"submission_dir":  "/path/to/payload" "Status": "INVALID",  "failed_checkpoints": {ONLY TEH FAILED ISSUES}}
{"submission_dir":  "/path/to/payload" "Status": "INVALID",  "failed_checkpoints": {ONLY TEH FAILED ONES}}
{"submission_dir":  "/path/to/payload" "Status": "INVALID",  "failed_checkpoints": {ONLY TEH FAILED ONES}}
{"submission_dir":  "/path/to/payload" "Status": "INVALID",  "failed_checkpoints": {ONLY TEH FAILED ONES}}
}
```
This way I can action on fixing all of the things without going into individual folders for each one, and rerun validation in bulk. 

3. The tool is too opinionated on the data formatting for input.  It is not easily usable as it is not easily scriptable as you have to conform to the directory structure.  I had to rename all my payloads as I moved them to the location. This was annoying, and I don’t think it added value. I had them named to align with my data (similar to how Hardeep names the import payloads with the EGA id) to be informational to me. It's possible to yes, but it feels much too strict.  It also introduces room for error, as when you are looking for files they all look the same; when looking through possibly a lot of failing payloads this has a large potential to be confusing and hard to follow. 
- [x] It would be easier if the command was this: seq-tools validate  /path/to/payload/filename  /directory/to/files 
- [x] remove the restriction that payload has to be named `sequencing_experiment.json`

4. Having a file in a submission directory causes a weird error. 
```
rbajari@wl5503-rbajar:~/DASH-CA_transferred_data/testing-payloads$ seq-tools validate *.*
Traceback (most recent call last):
  File "/home/rbajari/.local/bin/seq-tools", line 11, in <module>
    sys.exit(main())
  File "/home/rbajari/.local/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/rbajari/.local/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/rbajari/.local/lib/python3.6/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/rbajari/.local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/rbajari/.local/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/rbajari/.local/lib/python3.6/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/rbajari/.local/lib/python3.6/site-packages/seq_tools/cli/__init__.py", line 89, in validate
    perform_validation(ctx, subdir=subdir)
  File "/home/rbajari/.local/lib/python3.6/site-packages/seq_tools/validation/__init__.py", line 55, in perform_validation
    initialize_log(ctx, subdir)
  File "/home/rbajari/.local/lib/python3.6/site-packages/seq_tools/utils.py", line 48, in initialize_log
    os.mkdir(log_directory)
NotADirectoryError: [Errno 20] Not a directory: 'report.json/logs'
```

###  Optional Changes: 
5. On the initial output, some color coding would be nice.  If it is invalid, highlighting it in red would be nice. 

![image](https://user-images.githubusercontent.com/15824252/101186450-1a6b6a80-3621-11eb-9659-a092d201f474.png)

6. Some issues with python2 v python 3 will exist in the instructions.  For example, I had to use pip3 to install samtools.  This is obviously system dependent. It would be better if we could package this with dependencies in a way that does not require the user to do this extra work to install. Maybe as a Docker or PyInstaller?

**Comment by junjun-zhang** [comment]
closing this as all items being checked, plus we now have different color for PASS, INVALID etc, and Docker as well
